File: app/src/main/java/io/github/v2compose/util/RefererUtils.java
package io.github.v2compose.util;


import io.github.v2compose.network.NetConstants;



public interface RefererUtils {

    String TINY_REFER = NetConstants.BASE_URL + "/mission/daily";

    public static String topicReferer(String topicId) {
        return NetConstants.BASE_URL + "/t/" + topicId;
    }

    public static String userReferer(String username) {
        return NetConstants.BASE_URL + "/member/" + username;
    }
}

File: shared/src/commonMain/kotlin/io/github/v2compose/utils/RefererUtils.kt
package io.github.v2compose.utils

import io.github.v2compose.network.NetConstants

object RefererUtils {
    val TINY_REFER: String = NetConstants.BASE_URL + "/mission/daily"

    fun topicReferer(topicId: String?): String {
        return NetConstants.BASE_URL + "/t/" + topicId
    }

    fun userReferer(username: String?): String {
        return NetConstants.BASE_URL + "/member/" + username
    }
}
